support generating sql statements with the proper formatting for tabl…

jwong92 opened the pull request:
…e names

previously double quotes was always being added to table names, and as such was not properly formatting namespaced tables.
We found a class method on PG that supports adding quotes to a string, so separated the previously manual method into two
separate methods that would allow for our use case

bradrobertson commented on settings/view_data_pg.json:
```diff
@@ -3,7 +3,7 @@
   "hostaddr": "127.0.0.1",
   "port": 5432,
-  "user": "view_data",
+  "user": "jwong",
```
remove

bradrobertson commented on test/test_init.rb:
```diff
@@ -7,6 +7,7 @@
 
 require 'test_bench'; TestBench.activate
 
+require 'pry'
```
we can probably remove this dev dep as part of this MR (and the gemspec changes)

jwong92 committed:
generate sql statements with the proper formatting for table

previously double quotes was always being added to table names, and as such was not properly formatting namespaced tables.
We found a class method on PG that supports adding quotes to a string, so separated the previously manual method into two
separate methods that would allow for our use case
